### DISCUSSION TITLE
I need 'd' to delete visual selection

### DISCUSSION THREAD

**Discussion by lnxfsf** [vscode-neovim/vscode-neovim#1272] [Q&A]
I need 'd' to delete visual selection, how I can do that ?

I tried 
```
        
{
  "key": "d",
  "command": "vscode-neovim.send",
  "when": "editorTextFocus && neovim.mode == 'visual' ",
  "args": "d"
}

```

But it doesn't work.

while this works
```
    {
        "key": "backspace",
        "command": "vscode-neovim.send",
        "when": "editorTextFocus && neovim.mode == 'visual'",
        "args": "backspace"
        }

```

I need same behaviour I have for backspace, but for 'd', as in Vim, 'd' for selection deletes that selection

**Comment by theol0403** [comment]
Why does this not work by default? What kind of visual selection?

You can't bind alpha `type` keys in vscode.

**Reply by lnxfsf** [reply]
> What kind of visual selection?


visual selection from VSCode. 
And then 'd', should delete it. 



> Why does this not work by default?

I don't know 

In nvim, it works fine. as expected, but in VSCode, nothing..
